test/message/promise_fast_handled_reject.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+
+const p1 = new Promise(function(res, rej) {
+  consol.log('One'); // eslint-disable-line no-undef
+});
+
+new Promise(function(res, rej) {
```

Comment:
This test _can_ imply that the reason `'Two'` is the output is because there is no `const` reference to it. What happens when you do `const p2 = new Promise(...`?